aggregator.py

```diff
@@ -605,6 +617,10 @@ def __init__(self, hostname, interval=1.0, expiry_seconds=300, formatter=None, r
             'ms': Histogram,
             's': Set,
         }
+        self.histogram_aggregates = histogram_aggregates\
+           or get_histogram_aggregates()
```

Comment:
Why do we need to recall this function ? Hasn't it already be done when initiating the config ?